functions/tech-support

```diff
@@ -314,257 +331,299 @@ if cli-shell-api existsActive protocols bgp; then
+  if cli-shell-api existsActive service dhcpv6-server; then
+    header DHCPV6 Server Status
+    show dhcpv6 server status 
+    header DHCPV6 Server Leases
+    show dhcpv6 server leases
+  else
+    echo "DHCPV6 Server is not configured"
+  fi
 
-header DHCPV6 Relay 
-if cli-shell-api existsActive service dhcpv6-relay; then
-  header DHCPV6 Relay Status
-  show dhcpv6 relay-agent status 
-else
-  echo "DHCPV6 Relay is not configured"
-fi
+  header DHCPV6 Relay 
+  if cli-shell-api existsActive service dhcpv6-relay; then
+    header DHCPV6 Relay Status
+    show dhcpv6 relay-agent status 
+  else
+    echo "DHCPV6 Relay is not configured"
+  fi
 
-header DHCPV6 Client
-is_dhcp=`show dhcpv6 client leases | wc -l`
-if [ "$is_dhcp" == "1" ]; then
-echo "DHCPV6 client is not configured"
-else
-header DHCPV6 Client Leases 
-show dhcpv6 client leases
-fi
+  header DHCPV6 Client
+  is_dhcp=`show dhcpv6 client leases | wc -l`
+  if [ "$is_dhcp" == "1" ]; then
+  echo "DHCPV6 client is not configured"
+  else
+  header DHCPV6 Client Leases 
+  show dhcpv6 client leases
+  fi
 
-header DNS 
-if cli-shell-api existsActive service dns; then
-  header DNS Dynamic Status
-  show dns dynamic status
-  header DNS Forwarding Statistics
-  show dns forwarding statistics
-  header DNS Forwarding Nameservers  
-  show dns forwarding nameservers
-else
-  echo "DNS is not configured"
-fi
+}
 
+function show_dns { 
+  header DNS 
+  if cli-shell-api existsActive service dns; then
+    header DNS Dynamic Status
+    show dns dynamic status
+    header DNS Forwarding Statistics
+    show dns forwarding statistics
+    header DNS Forwarding Nameservers  
+    show dns forwarding nameservers
+  else
+    echo "DNS is not configured"
+  fi
+}
 
-header FIREWALL 
-if cli-shell-api existsActive firewall; then
-  header Firewall Group
-  show firewall group
-  header Firewall Summary
-  show firewall summary
-  header Firewall Statistics
-  show firewall statistics 
-else
-  echo "Firewall is not configured"
-fi
+function show_additional_network { 
+  header FIREWALL 
+  if cli-shell-api existsActive firewall; then
+    header Firewall Group
+    show firewall group
+    header Firewall Summary
+    show firewall summary
+    header Firewall Statistics
+    show firewall statistics 
+  else
+    echo "Firewall is not configured"
+  fi
 
-header IPSec
-if cli-shell-api existsActive vpn ipsec; then
-  header IPSec Status
-  show vpn ipsec status
-  header IPSec sa
-  show vpn ipsec sa
-  header IPSec sa Detail
-  show vpn ipsec sa detail
-  header IPSec sa Statistics
-  show vpn ipsec sa statistics
-   
-  header /etc/ipsec.conf
-  cat /etc/ipsec.conf
-
-  if [ -r /etc/ipsec.secrets ]; then
-    header /etc/ipsec.secrets
-    cat /etc/ipsec.secrets
+  header IPSec
+  if cli-shell-api existsActive vpn ipsec; then
+    header IPSec Status
+    show vpn ipsec status
+    header IPSec sa
+    show vpn ipsec sa
+    header IPSec sa Detail
+    show vpn ipsec sa detail
```

Comment:
This command no longer exists.

Reply:
Delete this command

Reply:
So you've kept all "run show vpn ipsec sa *" commands that no longer exist and removed the one that does? https://github.com/vyos/vyatta-op/blob/9d6c8e3426145818802fc62e75168925bc4323cf/functions/tech-support#L416-L422

Or I'm misunderstanding your intent?

Reply:
Please check .There was a just some misunderstanding, sorry